lib/ecs_deploy/service.rb

```diff
@@ -172,13 +173,23 @@ def self.wait_all_running(services)
         client = Aws::ECS::Client.new(region: region)
         ss.reject(&:delete).map(&:service_name).each_slice(MAX_DESCRIBE_SERVICES).map do |chunked_service_names|
           Thread.new do
+            last_events = {}
             EcsDeploy.config.ecs_wait_until_services_stable_max_attempts.times do
               EcsDeploy.logger.info "wait service stable [#{chunked_service_names.join(", ")}] [#{cl}]"
               resp = client.describe_services(cluster: cl, services: chunked_service_names)
               resp.services.each do |s|
                 # cf. https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-sdk-ecs/lib/aws-sdk-ecs/waiters.rb#L91-L96
                 if s.deployments.size == 1 && s.running_count == s.desired_count
                   chunked_service_names.delete(s.service_name)
+                else
+                  service = ss.detect { _1.cluster == File.basename(s.cluster_arn) && _1.service_name == s.service_name }
```

Comment:
The syntax `_1` is supported by Ruby 3.0 or later, and I think we should support at least Ruby 2.7 until its end-of-life.
cf. https://endoflife.date/ruby